simulations/vip-433/arbitrumone.ts

```diff
@@ -0,0 +1,134 @@
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { formatUnits, parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { setMaxStalePeriodInChainlinkOracle } from "src/utils";
+import { forking, testForkedNetworkVipCommands } from "src/vip-framework";
+import ERC20_ABI from "src/vip-framework/abi/erc20.json";
+import { checkIsolatedPoolsComptrollers } from "src/vip-framework/checks/checkIsolatedPoolsComptrollers";
+import { checkRiskParameters } from "src/vip-framework/checks/checkRiskParameters";
+import { checkVToken } from "src/vip-framework/checks/checkVToken";
+import { checkInterestRate } from "src/vip-framework/checks/interestRateModel";
+
+import vip433, {
+  COMPTROLLER_CORE,
+  REFUND_ADDRESS,
+  REFUND_AMOUNT,
+  REFUND_TOKEN,
+  market,
+  token,
+} from "../../vips/vip-433/bscmainnet";
+import POOL_REGISTRY_ABI from "./abi/PoolRegistry.json";
+import RESILIENT_ORACLE_ABI from "./abi/ResilientOracle.json";
+import COMPTROLLER_ABI from "./abi/comptroller.json";
+import VTOKEN_ABI from "./abi/vToken.json";
+
+const BLOCKS_PER_YEAR = BigNumber.from("31536000");
+const ONE_YEAR = 3600 * 24 * 365;
+
+const { POOL_REGISTRY, NORMAL_TIMELOCK, RESILIENT_ORACLE, CHAINLINK_ORACLE } = NETWORK_ADDRESSES["arbitrumone"];
```

Comment:
nit: generally we use hardcoded addresses to freeze the variables used in the VIP in the event they might be affected by a package update